doc/classes/Control.xml

```diff
@@ -962,6 +969,27 @@
 		</method>
 	</methods>
 	<members>
+		<member name="accessibility_controls_nodes" type="NodePath[]" setter="set_accessibility_controls_nodes" getter="get_accessibility_controls_nodes" default="[]">
+			The list of nodes which are controlled by this node.
```

Comment:
This whole "control/controlled" thing is quite puzzling with Control nodes in mind.